base/docs/basedocs.jl

```diff
@@ -925,13 +925,27 @@ Short-circuiting boolean AND.
 """
 kw"&&"
 
+"""
+    `and` is not used in Julia
+
+Use `&&` for short-circuiting boolean AND.
+"""
+kw"and"
+
 """
     x || y
 
 Short-circuiting boolean OR.
 """
 kw"||"
 
+"""
+    `or` is not used in Julia
+
+Use `|| for short-circuiting boolean OR.
```

Comment:
is there a backtick missing here?

Reply:
can't believe I just did that.... thx for catching it